bpmn/Dockerfile

```diff
@@ -26,13 +26,19 @@ RUN /usr/local/bin/pkg --targets ${NODE}-${PLATFORM}-${ARCH} . -o app.bin
 # Create the image
 FROM alpine:3.13.2
 
-COPY --from=builder /app/app.bin /node/bin/bpmn
-COPY assets /app/assets
+RUN apk add --no-cache --update chromium
+
+RUN addgroup -g 1000 kroki && adduser -D -G kroki -u 1000 kroki
+
+USER kroki
+WORKDIR /home/kroki
+
+COPY --from=builder /app/app.bin /usr/bin/bpmn
+COPY --chown=kroki:kroki assets /app/assets
```

Comment:
Where should `assets` be placed? Keep `/app/assets`?

Reply:
I think `/usr/local/kroki/assets` would be fine, so we know where to look for Kroki related files on all Docker images.
Don't forget to update `KROKI_BPMN_PAGE_URL`.